README.md

```diff
@@ -1,44 +1,25 @@
 # VA.gov Team Repository
-*Last updated: [9/17/2019](#document-history)*
 
 This project aims to improve the lives of Veterans, their families, and their caregivers by facilitating the creation of interactive experiences on VA.gov on par with the private sector. It's comprised of a platform (the Veteran-facing Services Platform, aka VSP) which serves as the common framework for building cohesive products on VA.gov, and the VA.gov products themselves (the Veteran Facing Services, aka VFSs).
 
-This repository is used to store *non-code* documents and files created and used by teams building the VSP and VFSs.
+This repository is used to store *non-code* documents and files created and used by teams building the VSP and VFSs. It is also used for issue tracking for many parts of the VSP platform.
 
 ***An important note about security for contributors to this repository: this is a public, open source space! Which means it's great for collaborating, but decidedly not great for storing sensitive information (like credentials, PII, etc). So be mindful as you work in this repo. Don't store anything sensitive here!***
 
 For information on how to use this repo, see the [Repo Guidelines](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/orientation/repo-guidelines.md).
 
-#### Top level stats:
-
-- Number of VA.gov features live: 30something?
-- Number of VA.gov features in progress: 13?
-- Number of VFS Teams actively building VFSs: 13
-- Number of unique VA.gov user accounts: 2,984,036
-- VSP-hosted VA.gov page loads occuring in < 5s: 88.6%
-- Average uptime last 30 days: 99.1%
-
----------------
-
 ## How to navigate this repository
 
+### Knowledge hub
+The best starting place is the knowledge hub, that provides links to the content in this repository and elsewhere:
+https://department-of-veterans-affairs.github.io/va.gov-team/
+
 ### [/platform](./platform)
-All resources, templates, guides, and processes that support development of VFSs on VA.gov.
+All VSP resources, templates, guides, and processes that support development of VFSs on VA.gov.
 
 ### [/products](./products)
-All VFSs (aka Va.gov features and products)
+All VFSs (aka VA.gov features and products)
 
 ### [/teams](./teams)
-All teams currently building VSP or building VFSs.
-
----------------
-
-## Document history
```

Comment:
Is there a reason you deleted the document history?

Reply:
We don't have a formal stance on this (doc-level change summaries). It's nice to see changes at a glance (particularly because the GH history can be inaccurate — migrated files are shown as created by the person who migrated, and renamed files or files that move folders are shown as new upon renaming/moving), but we don't recommend one way or the other currently on VSP. On this readme — which does not have another "owner" or "point of contact" it also serves as a quick way to show who's been making significant updates. But again, in the absence of a formal stance/process here, I could go either way.